src/player-config.js

```diff
@@ -59,8 +59,7 @@ const DefaultConfig = {
     ]
   },
   streaming: {
-    forceBreakStall: false,
-    lowLatencyMode: true
+    forceBreakStall: false
```

Comment:
why not lowLatencyMode: false ?

Reply:
https://shaka-player-demo.appspot.com/docs/api/shaka.extern.html#.StreamingConfiguration
it is false by default

Reply:
@JonathanTGold 
please add 
lowLatencyMode: false 

we must merge it  ASAP

Reply:
Right ,  and it's false by default in shaka, everyone keeps its default.